cli/config/configfile/file.go

```diff
@@ -36,6 +36,10 @@ type ConfigFile struct {
 	Filename             string                      `json:"-"` // Note: for internal use only
 	ServiceInspectFormat string                      `json:"serviceInspectFormat,omitempty"`
 	ServicesFormat       string                      `json:"servicesFormat,omitempty"`
+	HTTPProxy            string                      `json:"httpProxy,omitempty"`
+	HTTPSProxy           string                      `json:"httpsProxy,omitempty"`
+	NoProxy              string                      `json:"noProxy,omitempty"`
+	FTPProxy             string                      `json:"ftpProxy,omitempty"`
```

Comment:
Should the json keys be all uppercase to match what people would have to type on the --build-arg option? Since case does matter (at least in some envs) I think using the right variant would reduce potential confusion and typos.